provision/setup-montagu

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env bash
```

Comment:
This is based off of an restore script that was never really used. But with the vault setup now included we can just go ahead and set everything up